### PR TITLE
DAOS-2526 doc: create daos tool man-page

### PR DESCRIPTION
**Pull request by bfaccini**
This is basically a reformat of daos command
on-line help in a man-page ...

Change-Id: I24e682af033a54d1cf367723cf6afcc04890c771
Signed-off-by: Bruno Faccini <bruno.faccini@intel.com>

### CODE REVIEW

File: doc/man/man8/daos.8 [hunk 1]
```diff
@@ -0,0 +1,172 @@
+.\" (C) Copyright 2015, 2016, 2017 Intel Corporation.
```
Comment (cdurf1) [inline doc/man/man8/daos.8]: Copyright 2015 - 2019

File: doc/man/man8/daos.8 [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+.TH daos 8 2019-09-17 "0.0.1" "DAOS Commands"
+.SH NAME
+daos \- command to manage DAOS pool/container/object
+.SH SYNOPSIS
+.br
+.B daos RESOURCE COMMAND \fR[\fBOPTION\fRs]
+.SH DESCRIPTION
+.B daos
+can be used to manage/query a pool content, create/query/manage/destroy a
+container inside a pool, query/manage an object inside a container.
```
Comment (cdurf1) [inline doc/man/man8/daos.8]: ...pool, or query/.....

File: doc/man/man8/daos.8 [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	  \fB--force\fR            destroy container regardless of state
+.TP
+.I container \fBOPTION\fRs (query, and all commands except create):
+	  <\fIpool\fR options>   with \fB--cont\fR use: (\fB--pool\fR, \fB--sys-name\fR, \fB--svc\fR)
+.br
+	  <\fIpool\fR options>   with \fB--path\fR use: (\fB--sys-name\fR, \fB--svc\fR)
+.br
+	  \fB--cont=\fRUUID        (mandatory, or use \fB--path\fR)
+.br
+	  \fB--path=\fRPATHSTR     (mandatory, or use \fB--cont\fR)
```
Comment (bfaccini) [inline doc/man/man8/daos.8]: I should also expand a little bit here too!

File: doc/man/man8/daos.8 [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+.I container \fBOPTION\fRs (create and link to namespace path):
+	  <\fIpool\fR/\fIcont\fR opts>   (\fB--pool\fR, \fB--sys-name\fR, \fB--svc\fR, \fB--cont\fR [optional])
+.br
+	  \fB--path=\fRPATHSTR     container namespace path to be created and provide a direct link to new DAOS container
+.br
+	  \fB--type=\fRCTYPESTR    container type (HDF5, POSIX)
+.br
+	  \fB--oclass=\fROCLSSTR   container object class
+.br
+				      (tiny, small, large, R2, R2S, repl_max)
```
Comment (mchaarawi) [inline doc/man/man8/daos.8]: we should update those to what is supported today:
(ex: S{1,2,4,X}, RP_2G{1,2,4,X}, RP_3G{1,2,4,X}, RP_4G{1,2,4,X}, RP_XSF)

Comment (bfaccini) [reply doc/man/man8/daos.8]: Done, but what do all these acronyms mean ??

**Pull request merged**